src/actions/nodes.js

```diff
@@ -70,7 +70,7 @@ export function addNodeMetadata(data) {
 
 /**
  * update node metadata on selection, loading new data if it has not been previously called
- * @param {string} nodeID node id of clicked node
+ * @param {String} nodeID node id of clicked node
  * @return {function} A promise that resolves when the data is loaded
```

Comment:
```suggestion
 * @return {Function} A promise that resolves when the data is loaded
```
I think these should also be changed; see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function.